make/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "make.js",
+  "version": "1.0.0",
+  "description": "",
+  "main": "make.js",
+  "scripts": {
+    "test": "mocha"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
```

Comment:
Why do you use `package.json` several times in your project?
It should be included once in the root of the branch.
Also about linter config